scripts/unify-radius.cjs

```
@@ -0,0 +1,84 @@
+  }
+  // directional (e.g. 4px 0 0 4px) -> keep zeros, route rounded corners
+  return tokens.map((tok) => (isZero(tok) ? '0' : isLength(tok) ? RADIUS : tok)).join(' ');
+};
+
+const RE = /border-radius:\s*var\((--[\w-]+),\s*([^)]+?)\)/g;
+
+const walk = (dir) => {
+  const out = [];
+  for (const entry of fs.readdirSync(dir, { withFileTypes: true })) {
+    const full = path.join(dir, entry.name);
+    if (entry.isDirectory()) out.push(...walk(full));
```

ESLint

[error] Expected { after 'if' condition.

(curly)